django_ibmi/creation.py

```diff
@@ -36,10 +28,7 @@
 from django import VERSION as djangoVersion
 from django.db.backends.utils import truncate_name
 
-if _IS_JYTHON:
-    dbms_name = 'dbname'
-else:
-    dbms_name = 'dbms_name'
+dbms_name = 'dbms_name'
```

Comment:
Since there's only one option now, this variable should get inlined. Of course, I don't think it will work anyway, because pyodbc doesn't have a dbms_name property.